src/Smidge/Controllers/AddExpiryHeadersAttribute.cs

```diff
@@ -13,21 +13,26 @@ namespace Smidge.Controllers
     /// </summary>
     public sealed class AddExpiryHeadersAttribute : Attribute, IFilterFactory, IOrderedFilter
     {
-        public IFilterMetadata CreateInstance(IServiceProvider serviceProvider) => new AddExpiryHeaderFilter(serviceProvider.GetRequiredService<IHasher>(), serviceProvider.GetRequiredService<IBundleManager>());
+        public IFilterMetadata CreateInstance(IServiceProvider serviceProvider) => new AddExpiryHeaderFilter(
+            serviceProvider.GetRequiredService<ISmidgeProfileStrategy>(),
+            serviceProvider.GetRequiredService<IHasher>(),
+            serviceProvider.GetRequiredService<IBundleManager>());
 
         public bool IsReusable => true;
 
         public int Order { get; set; }
 
         public sealed class AddExpiryHeaderFilter : IActionFilter
         {
+            private readonly ISmidgeProfileStrategy _profileStrategy;
             private readonly IHasher _hasher;
             private readonly IBundleManager _bundleManager;
 
-            public AddExpiryHeaderFilter(IHasher hasher, IBundleManager bundleManager)
+            public AddExpiryHeaderFilter(ISmidgeProfileStrategy profileStrategy, IHasher hasher, IBundleManager bundleManager)
```

Comment:
There's a few ctor changes on public classes which would be breaking changes requiring a major version rev. Can we avoid this with overloads? Else I think this would technically need a new major version